api/src/test/java/uk/gov/dhsc/htbhf/claimant/converter/EligibilityOverrideToEligibilityOverrideDTOConverterTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.dhsc.htbhf.claimant.converter;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
+import uk.gov.dhsc.htbhf.claimant.model.EligibilityOverrideDTO;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static uk.gov.dhsc.htbhf.TestConstants.MAGGIE_AND_LISA_DOBS;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.EligibilityOverrideTestDataFactory.aConfirmedEligibilityOverrideWithChildren;
+
+class EligibilityOverrideToEligibilityOverrideDTOConverterTest {
+
+    private EligibilityOverrideToEligibilityOverrideDTOConverter converter = new EligibilityOverrideToEligibilityOverrideDTOConverter();
+
+    @Test
+    void shouldConvertEligibilityOverrideToEligibilityOverrideDTO() {
+        //Given
+        EligibilityOverride eligibilityOverride = aConfirmedEligibilityOverrideWithChildren(MAGGIE_AND_LISA_DOBS);
+
+        //When
+        EligibilityOverrideDTO result = converter.convert(eligibilityOverride);
+
+        //Then
+        assertThat(result).isNotNull();
+        assertThat(result.getEligibilityOutcome()).isEqualTo(eligibilityOverride.getEligibilityOutcome());
+        assertThat(result.getChildrenDob()).isEqualTo(MAGGIE_AND_LISA_DOBS);
+        assertThat(result.getOverrideUntil()).isEqualTo(eligibilityOverride.getOverrideUntil());
```

Comment:
That's good, but I also meant for the other two assertions too, so lets put in the actual value for the outcome and override until date too for clarity.